pkgs/development/python-modules/cloudflare/default.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, requests
+, future
+, pyyaml
+, jsonlines
+}:
+
+buildPythonPackage rec {
+  pname = "cloudflare";
+  version = "2.6.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "4463d5f2927338384169315f34c2a8ac0840075b59489f8d1d773b91caba6c39";
+  };
+
+  propagatedBuildInputs = [
+    requests
+    future
+    pyyaml
+    jsonlines
+  ];
+
+  # no tests associated with package
+  doCheck = false;
```

ensure it's not trivially broken
```suggestion
  doCheck = false;
  pythonImportsCheck = [ "CloudFlare" ];
```

---

Fixed.